cliquet/initialization.py

```diff
@@ -59,11 +66,10 @@ def _redirect_to_version_view(request):
             '/%s/%s' % (route_prefix, request.matchdict['path']))
 
     # Disable the route prefix passed by the app.
-    route_prefix = config.route_prefix
     config.route_prefix = None
 
     config.add_route(name='redirect_to_version',
-                     pattern='/{path:(?!%s).*}' % route_prefix)
+                     pattern='{path:(?!v[0-9]+).*}')
```

Comment:
The route doesn't starts with a /
They were a bug in the previous regexp. 

Actually it was a catch all. Now it only catches if it doesn't starts with a v.

this makes me thing that we should probably add a ``^`` at the beginning to make sure we matches the right ``v[0-9]+``